Common/Configurations/EnvironmentDescription.cs

```diff
@@ -54,11 +54,6 @@ void Dispose(bool disposing)
             }
         }
 
-        public bool SettingExists(string settingName)
```

Comment:
what is Setting ? We have another version of get/set setting - should we keep this as well ?